CHANGELOG.asciidoc

```diff
@@ -1245,6 +1245,7 @@ TinkerPop 3.0.0.M2 (Release Date: September 23, 2014)
 * Moved `GiraphGraph.getOutputGraph()` to `GiraphHelper`.
 * Changed `GIRAPH_GREMLIN_HOME` to `GIRAPH_GREMLIN_LIB` to reference directory where jars are to be loaded.
 * Updated README with release instructions.
+* if there is no edge label in the GraphML file, then use Edge.DEFAULT
```

Comment:
Why is this in the MX release section? Also, please capitalize "If" like all other CHANGELOG update.